include/clang/Basic/Attr.td

```diff
@@ -1080,6 +1080,12 @@ def ObjCBridgeRelated : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+def NSErrorDomain : Attr {
+  let Spellings = [GNU<"ns_error_domain">];
+  let Documentation = [Undocumented];
```

Comment:
Please add some documentation here, otherwise it won't get documented *anywhere*.